### ISSUE TITLE
Changelog web page not updated

### ISSUE THREAD

**Issue by antodc** [doublecmd/doublecmd#429]
Just to report that the current 1.0.5 version is not included in the changelog list here
https://doublecmd.sourceforge.io/mantisbt/changelog_page.php .

Thanks for the amazing work!

**Comment by alexx2000** [comment]
Yes, because GitHub [milestones ](https://github.com/doublecmd/doublecmd/milestone/5?closed=1)used instead.
